python-avd/pyavd/_eos_designs/structured_config/underlay/loopback_interfaces.py

```diff
@@ -76,9 +77,13 @@ def loopback_interfaces(self: AvdStructuredConfigUnderlayProtocol) -> None:
                 )
                 or None,
                 shutdown=False,
-                ip_address=f"{self.shared_utils.vtep_ip}/32",
             )
 
+            if self.shared_utils.underlay_ipv6_numbered:
+                vtep_loopback.ipv6_address = f"{self.shared_utils.vtep_ipv6}/{self.inputs.fabric_ip_addressing.loopback.ipv6_prefix_length}"
+            else:
+                vtep_loopback.ip_address = f"{self.shared_utils.vtep_ip}/32"
+
             if self.shared_utils.network_services_l3 is True and self.inputs.vtep_vvtep_ip is not None:
```

Comment:
I guess we should also exclude vvtep from support. So add to the list of limitations and move this `if` under the `else` above (so just indent this block)